neqo-transport/src/connection.rs

```diff
@@ -381,6 +381,35 @@ impl StateSignaling {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum QuicVersion {
+    Draft27,
+    Draft28,
+}
+
+impl QuicVersion {
+    pub fn as_u32(self) -> Version {
+        match self {
+            Self::Draft27 => 0xff00_0000 + 27,
+            Self::Draft28 => 0xff00_0000 + 28,
+        }
+    }
+}
+
+impl TryFrom<Version> for QuicVersion {
+    type Error = Error;
+
+    fn try_from(ver: Version) -> Res<Self> {
+        if ver == 0xff00_0000 + 27 {
```

Comment:
`match` ?  Or does the extra arithmetic fail here?

Reply:
yeah doesn't work with `match`.